templates/server/src/main/resources/archetype-resources/pom.xml

```diff
@@ -10,6 +10,9 @@
   <description>Application based on the Open Application Standard Platform for Java (devon4j).</description>
 
   <properties>
+    <revision>0.0.1</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <projectVersion>${revision}${changelist}</projectVersion>
```

Comment:
It is good that we also provide this as best practice to our projects via archetype. However some concerns:
* you are so fat just setting a property `projectVersion` but this is not related to the actual `${project.version}` of that maven project. IMHO you need to change this in the `version` tag and we have to test how `maven-archetype` mechanism works with this as it is usually setting a concrete version in the `pom.xml` files that can be provided on the commandline on archetype generation
* many people will never have heared about this feature and are clueless about the benefits and how to use it properly. We should therefore create a small guide in devon4j and link the ci-friendly maven documentation from there.
* then we should add an XML comment to that guide here

Reply:
BTW: To get started, you can just create an almost empty guide in the wiki and add the CI-friendly link and a TODO. We can update the wiki page later. Also I can contribute to that as well. For the comment use the link to the official wiki instead of the `devonfw-wiki` one.

Reply:
Wiki won't be required as of now. As the CI friendly maven versions for archetype will not be implemented as part of this feature.